lib/cartopy/tests/crs/test_equidistant_conic.py

```diff
@@ -23,12 +24,20 @@ def test_default(self):
                       'y_0=0.0', 'lat_1=20.0', 'lat_2=50.0'}
         check_proj_params('eqdc', eqdc, other_args)
 
-        assert_almost_equal(np.array(eqdc.x_limits),
-                            (-22784919.35600352, 22784919.35600352),
-                            decimal=7)
-        assert_almost_equal(np.array(eqdc.y_limits),
-                            (-10001965.729313632, 17558791.85156368),
-                            decimal=7)
+        if pyproj.__proj_version__ >= '9.2.0':
+            assert_almost_equal(np.array(eqdc.x_limits),
+                                (-22784919.3559981,  22784919.3559981),
+                                decimal=7)
+            assert_almost_equal(np.array(eqdc.y_limits),
+                                (-10001965.72931272,  17558791.85157471),
+                                decimal=7)
```

Comment:
I would restructure these blocks to something like this to avoid the duplicated `assert_almost_equal`s.
```suggestion
        expected_x = ...
        expected_y = ...
        if pyproj.__proj_version__ >= '9.2.0':
            expected_x = ...
            expected_y = ...
        assert_array_equal(eqdc.x_limits, expected_x)
        assert_array_equal(eqdc.y_limits, expected_y)
```

Reply:
done